easyaudit/signals/model_signals.py

```diff
@@ -62,14 +62,16 @@ def pre_save(sender, instance, raw, using, update_fields, **kwargs):
                 # We need a better way for this to work. ManyToMany will fail on pre_save on create
                 return None
 
-            if instance.pk is None:
-                created = True
-            else:
+            # Query the object, see if it exists
+            try:
+                old_model = sender.objects.get(id=instance.pk)
```

Comment:
Please change this to `pk=instance.pk`

I have primary keys not named `id` so this throws an exception

Reply:
@steverecio updated